src/cantools/database/__init__.py

```diff
@@ -258,6 +260,7 @@ def load(fp: TextIO,
          frame_id_mask: Optional[int] = None,
          prune_choices: bool = False,
          strict: bool = True,
+         best_effort: bool = False,
```

Comment:
Breaks this call:

https://github.com/Huxelerate/cantools/blob/d4eb11b56a3c2d2fe039f3586116f1af09eb2113/src/cantools/database/__init__.py#L88-L93